cfme/common/provider.py

```diff
@@ -92,6 +92,13 @@ def __init__(self, **kwargs):
                 self.token = kwargs['token']
 
     @property
+    def properties_form(self):
```

Comment:
@jkrocil if I change the name to properties_region, are you still against this functionality going in at the base class?